.gitignore

```diff
@@ -13,3 +13,7 @@ virtualenv/
 *.egg-info/
 build/
 twistd.log
+
+# NodeJS
+node_modules/
```

Comment:
`node_modules` is some kind of `site-packages` from node.

We will see how we integrate with nodeenv ... we might want to have everything inside a single
`venv` directory  where we store the python venv and where we will also have `venv/node_modules`

Reply:
I tried to have everything installed in a virtualenv with `-g`, but it looks like Serverless plugins [don't work from a global install](https://forum.serverless.com/t/plugins-dont-work-when-installed-globally/5874). So we will have the `node_modules` folder if we use it.

Reply:
ok. no problem. we can live with node_modules ignored in the root... this is life :) 

but when possible we should prefer a nodeenv and -g